.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Oops, I think I committed this file by accident. I think it is just some setting from the VSCode project I am using. Will remove in next commit.

Reply:
don't forget to remove this...

Reply:
Ha! I did remove it, but I think I accidentally recommitted it again :P. Oh well, I will add it to `.gitignore` so I don't do it again.